packages/by-name/mkNixosConfig/package.nix

```diff
@@ -35,15 +35,24 @@ lib.makeOverridable (
 
       # TODO(katexochen): imporve, see comment above.
       nixpkgs.overlays = [
-        (_self: _super: {
-          inherit (outerPkgs)
-            azure-no-agent
-            cloud-api-adaptor
-            kernel-podvm-azure
-            pause-bundle
-            ;
-          inherit (outerPkgs.kata) kata-agent;
-        })
+        (
+          _self: _super:
+          {
+            inherit (outerPkgs)
+              azure-no-agent
+              cloud-api-adaptor
+              kernel-podvm-azure
+              pause-bundle
+              ;
+            inherit (outerPkgs.kata) kata-agent;
+          }
+          // lib.optionalAttrs config.contrast.gpu.enable {
```

Comment:
Do we need to make the packages conditional on the `gpu.enable`? Isn't everything here lazily evaluated anyway?

Reply:
@msanft Did we resolve this offline? I can't remember.

Reply:
Sorry, forgot to push :P